File: 8KYU/Grasshopper - Grade Book/solution.py
def get_grade(a, b, c):
    mean = (a + b + c) / 3
  
    if mean >= 90:
        return "A"    
    if mean >= 80:
        return "B"
    if mean >= 70:
        return "C";
    if mean >= 60:
        return "D";
    return "F"
    

File: 8KYU/Grasshopper - Grade Book/solution.cs
using System;

public class Kata
{
	public static char GetGrade(int s1, int s2, int s3)
	{
		var mean = (s1 + s2 + s3) / 3;
    
		if(mean >= 90)
		  return 'A';
		else if(mean >= 80)
		  return 'B';
		else if(mean >= 70)
		  return 'C';
		else if(mean >= 60)
		  return 'D';
		else  
		  return 'F';
	}
}